### DISCUSSION TITLE
Query Results Search Feature in DBeaver

### DISCUSSION THREAD

**Discussion by bharathbaiju** [dbeaver/dbeaver#35105] [Q&A] [answered]
I hope this message finds you well.

I am writing to inquire about a specific functionality within DBeaver. After running a query and obtaining the results, I would like to know if it is possible to search within those results. For instance, if I have a result set with 1000 rows, I would like to search for a specific keyword, such as “disable,” and quickly locate all instances where this word appears.

This feature would significantly enhance the efficiency of data analysis and management tasks.

I appreciate any guidance or instructions you can provide on this matter.

Thank you for your support.

**Comment by ShadelessFox** [comment, marked as answer]
Hello @bharathbaiju,

Have you tried using **Edit -> Find Replace** (or the shortcut **CTRL+F** while the grid is focused)?

![image](https://github.com/dbeaver/dbeaver/assets/35821147/4080cbe7-7f52-433c-8f06-e78d2050bfd3)

Please let me know if it solves your issue.
